apidaze/http/__init__.py

```diff
@@ -18,8 +18,11 @@ def __init__(self, api_key: str, api_secret: str):
         if not self.api_key or not self.api_secret:
             raise ValueError('api_key and api_secret must be provided')
 
+        # environment = 'https://api4.apidaze.io/'
```

Comment:
I think should use the production API URL even though we have some un-deployed new endpoints. We might assume those will be deployed in a reasonable amount of time.

@joea-vi do you have any opinion on this? Would you prefer not to expose the `/applications` endpoint related methods until they are deployed? Or is it fine for you if we just expose them even though they will fail on the production API?